sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/MathUtils.scala

```diff
@@ -26,33 +27,39 @@ object MathUtils {
 
   def addExact(a: Int, b: Int): Int = withOverflow(Math.addExact(a, b))
 
-  def addExact(a: Int, b: Int, errorContext: String): Int =
-    withOverflow(Math.addExact(a, b), hint = "try_add", errorContext = errorContext)
+  def addExact(a: Int, b: Int, errorContext: Option[QueryContext]): Int = {
```

Comment:
can we use a consistent naming `context`?

Reply:
done